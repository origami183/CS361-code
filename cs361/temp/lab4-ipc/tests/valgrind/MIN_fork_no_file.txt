==1910891== Memcheck, a memory error detector
==1910891== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1910891== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1910891== Command: ../ipc ../data/f4.txt
==1910891== 
CKSUM(../data/f4.txt) = ''
==1910891== 
==1910891== HEAP SUMMARY:
==1910891==     in use at exit: 0 bytes in 0 blocks
==1910891==   total heap usage: 3 allocs, 3 frees, 8,993 bytes allocated
==1910891== 
==1910891== All heap blocks were freed -- no leaks are possible
==1910891== 
==1910891== For lists of detected and suppressed errors, rerun with: -s
==1910891== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
