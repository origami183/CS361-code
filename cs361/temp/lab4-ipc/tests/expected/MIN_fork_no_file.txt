CKSUM(../data/f4.txt) = ''
